CondFormats/PPSObjects/interface/CTPPSPixelIndices.h

```diff
@@ -52,7 +52,7 @@
 
 */
 
-namespace {
+namespace RPixValues {
```

Comment:
"For namespaces use lower case, e.g. namespace edm."
2.7 in https://cms-sw.github.io/cms_coding_rules.html

Reply:
@tvami, I did not check and had 50% of doing right ;-)
Done.

Reply:
@fabferro thanks! haha!
Please squash the two commits, thanks!